ampl-converter/src/test/java/com/powsybl/ampl/converter/AbstractAmplExporterTest.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.ampl.converter;
+
+import com.powsybl.commons.datasource.MemDataSource;
+import com.powsybl.commons.test.AbstractSerDeTest;
+import org.junit.jupiter.api.BeforeEach;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import static com.powsybl.commons.test.ComparisonUtils.assertTxtEquals;
+
+abstract class AbstractAmplExporterTest extends AbstractSerDeTest {
```

Comment:
Could you add the author name?

Reply:
Done. I put @geofjamg as author because he made the method `assertEqualsToRef`.

Reply:
You should be an author too!

Reply:
I have updated the signature. Thanks!